metrics.py

```diff
@@ -161,6 +158,11 @@ def parse_args():
     post_pipeline_parser.add_argument(
         '--diseases', required=False, metavar='<path>', type=str, help=(
             'Disease information, found in /diseases.'))
+    post_pipeline_parser.add_argument(
+        '--drugs', required=False, metavar='<path>', type=str, help=(
+            'ChEMBL dataset, found in ../../input/annotation-files'
+        )
+    )
```

Comment:
To keep in line with the existing definitions (or alternatively you could reformat them all if you prefer this way; I don't mind). Personally I feel this is a bit more compact while at the same time more readable

```suggestion
    post_pipeline_parser.add_argument(
        '--drugs', required=False, metavar='<path>', type=str, help=(
            'ChEMBL dataset, found in ../../input/annotation-files'))
```

Reply:
Also, I'm a bit confused; is `../../` still relative to the release root in the cloud storage?

Reply:
Yes. Addressed in `60c3d16`